src/config/user_config.rs

```diff
@@ -19,7 +19,7 @@ pub struct UserConfig {
 
 impl UserConfig {
     /// The default name for the configuration file in Cargo's home.
-    pub const FILE_NAME: &str = "liner.toml";
+    pub const FILE_NAME: &'static str = "liner.toml";
```

Comment:
New Clippy warning I guess, thanks!

Reply:
Yeah, this is a warning released as part of 1.74.0 (I believe). https://github.com/rust-lang/rust/issues/115010

Reply:
Thanks for the link. The de-sugaring surely is surprising!